app/views/spree/components/presentation/_hero.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="hero">
+  <div class="hero_container">
+    <div class="hero_left">
+      <%= render(
+        "spree/components/typography/hero_title",
+        content: "A free, open-source ecommerce platform that gives you complete control over your store."
+      ) %>
+      <%= render(
+        "spree/components/typography/subtitle",
+        content: "Running a high-volume online business isn’t easy – but it doesn’t have to be a headache, either."
+      ) %>
+    </div>
+    <div class="hero_right">
```

Comment:
I'd remove it for now if we are not using it. good idea anyway, maybe we can keep track of it with an issue?